docs/architecture/adr-template.md

```diff
@@ -6,12 +6,22 @@
 
 ## Status
 
-> A decision may be "proposed" if it hasn't been agreed upon yet, or "accepted"
-> once it is agreed upon. Once the ADR has been implemented mark the ADR as
-> "implemented". If a later ADR changes or reverses a decision, it may be marked
-> as "deprecated" or "superseded" with a reference to its replacement.
-
-{Deprecated|Declined|Accepted|Implemented}
+> An architecture decision is considered "proposed" when a PR containing the ADR
```

Comment:
I generally agree with this view. If we are in accord, we should also move the existing items that are under the "Proposed" section in the ToC (README.md).

Reply:
Are the historically proposed ADRs accepted though? Might be worthwhile to take a look through them sometime and decide what we should accept and what we should reject.

Also, we might want to go back over ADRs that were accepted several years ago but still not implemented and perhaps reject some of them due to lack of interest?

Reply:
Given that we have an explicit "Rejected" section, I was guessing probably so? But feel free to revert that file if you think it's a concern. 🙂

Reply:
Yes I'd prefer to revert that if you don't mind. I've opened a follow-up issue to review proposed ADRs that we don't intend on merging (#7812).